files/setup-010-veba-ui.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# Copyright 2019 VMware, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-2
+
+# Setup Knative UI
+
+set -euo pipefail
+
+echo -e "\e[92mSetting up VEBA UI RBAC ..." > /dev/console
+cat > /root/config/veba-ui-rbac.yaml << EOF
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: veba-ui
+  namespace: vmware-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: veba-ui
+  namespace: vmware-functions
+rules:
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["watch", "get", "list", "create", "update", "delete"]
+  - apiGroups: ["serving.knative.dev"]
+    resources: ["services"]
+    verbs: ["watch", "get", "list", "create", "update", "delete"]
+  - apiGroups: ["eventing.knative.dev"]
+    resources: ["triggers"]
+    verbs: ["watch", get", "list", "create", "update", "delete"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: veba-ui
+  namespace: vmware-functions
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: veba-ui
+subjects:
+- kind: ServiceAccount
+  name: veba-ui
+  namespace: vmware-system
+EOF
+
+kubectl apply -f /root/config/veba-ui-rbac.yaml
+
+echo -e "\e[92mSetting up VEBA UI Secret ..." > /dev/console
+kubectl -n vmware-system create secret generic veba-ui-secret \
+    --from-literal=VCENTER_FQDN=${VCENTER_SERVER} \
+    --from-literal=VCENTER_PORT=443 \
+    --from-literal=VCENTER_USER=${VCENTER_USERNAME_FOR_VEBA_UI} \
+    --from-literal=VCENTER_PASS=${VCENTER_PASSWORD_FOR_VEBA_UI} \
```

Comment:
Could this cause issues with special characters in the password? I.e. does OVF tool do some escaping or 1:1 passthrough of the string literals?